src/shared/cli/cli-args.js

```diff
@@ -53,7 +53,27 @@ const { argv } = yargs
     description: 'Write to dist folder',
     type: 'boolean'
   })
+  .option('onlyUseCache', {
+    description: 'Only use cache (no http calls)',
```

Comment:
`x` seems like a fine character, let's use that.

```suggestion
    alias: 'x',
    description: 'Only use cache (no http calls)',
```